src/amo/components/InstallWarning/index.js

```diff
@@ -0,0 +1,184 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import type { AppState } from 'amo/store';
+import { ADDON_TYPE_EXTENSION, UNINSTALLED, UNKNOWN } from 'core/constants';
+import translate from 'core/i18n/translate';
+import log from 'core/logger';
+import tracking from 'core/tracking';
+import { withExperiment } from 'core/withExperiment';
+import Notice from 'ui/components/Notice';
+import type { AddonType } from 'core/types/addons';
+import type { I18nType } from 'core/types/i18n';
+import type { InstalledAddon } from 'core/reducers/installations';
+import type { WithExperimentInjectedProps } from 'core/withExperiment';
+
+import './styles.scss';
+
+type Props = {|
+  addon: AddonType,
+|};
+
+type CouldShowWarningParams = {|
+  addonIsExtension: boolean,
+  addonIsRecommended: boolean | void,
+  experimentEnabled: $PropertyType<
```

Comment:
`experimentIsEnabled`?

Reply:
I agree that sounds better, but I took the name from `WithExperimentInjectedProps` at https://github.com/mozilla/addons-frontend/blob/2019.07.25/src/core/withExperiment.js#L11, and I wanted to retain consistency. Maybe that one should be renamed too? I guess I can do that in this patch as nothing else is currently using `withExperiment`, although it will increase the size of the patch. I think I'll do it anyway.

Reply:
It was more to be consistent with the other params in this function but yeah, maybe it's better to rename the attribute in the HOC too.